consensus/__main__.py

```diff
@@ -9,12 +11,14 @@
 
 def main():
     # Get data from config
-    config = ConfigParser('config/config.txt') # To run by pressing play in pycharm, use ../config/config.txt
+    config = ConfigParser('config/config.txt')  # To run by pressing play in pycharm, use ../config/config.txt
     consensus_table = config.prefix + config.consensus
     comments_table = config.prefix + config.comments
     molgenis_server = molgenis.Session(config.server)
     history_table = config.history
     previous_exports = config.previous
+    if type(previous_exports) != list:
```

Comment:
is this related to the history fix?

Reply:
No, but it is a bug we never found because we never had one history date since we implemented this. I tested this code using only one history date (because the number of dates is not relevant for the fix) and without this line, it will then fail.